etcdctl/ctlv3/command/global.go

```diff
@@ -51,10 +51,10 @@ type GlobalFlags struct {
 }
 
 type secureCfg struct {
-	cert   string
-	key    string
-	cacert string
-
+	cert               string
+	key                string
+	cacert             string
+	serverName         string
```

Comment:
SRV isn't part of TLS, it doesn't belong here

Reply:
i place the value in a new struct discoveryCfg, please help review again